CHAOSScon/2020EU/desc.md

```diff
@@ -12,6 +12,22 @@ Co-located with FOSDEM
 
 January 31, 2020
 
+## Registration
+
+Registration will open soon!
+
+### Diversity and Inclusion
+
+To encourage and increase the participation of a diverse community, we are pleased to offer waivers for registration based on diversity and need. To get a your registration waived, please contact [Georg Link](linkgeorg@gmail.com)
```

Comment:
For requesting a diversity access ticket, we will have a form with some basic questions. We need to create that form to ask why they want to participate and why they are asking for a ticket.

Reply:
Okay lets wait until form is ready than modify this and merge

Reply:
I updated the file to include the link to the form